src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -237,12 +245,11 @@ function ComposerWithSuggestions({
 
             // Indicate that draft has been created.
             if (commentRef.current.length === 0 && newComment.length !== 0) {
-                Report.setReportWithDraft(reportID, true);
+                setDraftStatusForReportID(reportID, true);
             }
-
             // The draft has been deleted.
-            if (newComment.length === 0) {
-                Report.setReportWithDraft(reportID, false);
+            else if (newComment.length === 0) {
```

Comment:
I don't think `else` is justified here, it just increase code complexity

Reply:
Yes I agree, thanks !!